fix: removed faulty haip uri scheme (#1155)
italia/eidas-it-wallet-docs: docs/en/remote-flow.rst

.. note::
  For the ``authorization_endpoint`` the use of universal links are preferred over custom url-schemes because, when properly configured using Assetlinks JSON for Android and Apple App Site Association for iOS, they provide enhanced security by reducing the risk of URL hijacking.

>   Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. The URL schemes openid4vp:// and haip-vp:// defined in `OPENID4VP`_ and `OPENID4VC-HAIP`_are supported to ensure interoperability.

Author: ```suggestion
  Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. The URL schemes ``openid4vp://`` and ``haip-vp://`` defined in `OPENID4VP`_ and `OPENID4VC-HAIP`_are supported to ensure interoperability.
```

>   Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. To ensure interoperability, support for custom URL schemes is also RECOMMENDED according to HAIP `OPENID4VC-HAIP`_, and in particular using the custom URL schemes ``openid4vp://`` and ``haip-vp://``.

Reviewer: The last sentence could be clarified because the `openid4vp:// `scheme is defined by OPENID4VP, while `haip-vp://` is defined by OPENID4VC-HAIP. you could rephrasing it as: 

`according to OPENID4VP and OPENID4VC-HAIP, and in particular using the custom URL schemes openid4vp:// and haip-vp://, respectively.`

>   Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. To ensure interoperability, support for custom URL schemes is also RECOMMENDED according to HAIP `OPENID4VC-HAIP`_, and in particular using the custom URL schemes ``openid4vp://`` and ``haip-vp://``.

Author: ```suggestion
  Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. The URL schemes openid4vp:// and haip-vp:// defined in `OPENID4VP`_ and `OPENID4VC-HAIP`_are supported to ensure interoperability.
```
